duckduckgo/Android
# Implement SearchBarView

Task/Issue URL: https://app.asana.com/0/488551667048375/1201794047916734/f

### Description
- Move KeyboardAwareEditText to common-ui.
- Move ic_close, omni_field_background drawables to common-ui.
- Create a reusable and pluggable SearchBarView within the common-ui module. This component is not used anywhere at the moment except on the App Component Design Preview.

### Steps to test this PR

1. Open App Components Design Preview
2. Go to Interactive
3. See Search Bar. Note that just shows the raw view and no clickable actions or events are being handled here. 
4. Confirm that no changes should happen in the Browse tab address bar.

### UI changes
![app-preview-search-bar](https://user-images.githubusercontent.com/2943941/153412638-cbdabf07-0dd5-4871-aeea-ff1cd8633c6f.png)
es
